docs/openapi/saved_objects/saved_objects.yml

```diff
@@ -520,6 +520,96 @@ paths:
                 - id: 67a9021c-c97e-4499-8150-9722ab44edd4
                   type: visualization
                   fields: ['title', 'fieldFormatMap']
+      responses:
+        '200':
+          description: The bulk get request is successful.
+          content:
+            application/json:
+              schema:
+                type: object  
+  /api/saved_objects/_export:
```

Comment:
can we change the PR title to match the API we are documenting?

Reply:
Sure let me make change of the title